state_models/vehicle_state.py

```diff
@@ -0,0 +1,66 @@
+from state_models.dynamic_module import Dynamic_module
```

Comment:
Not the biggest fan of this naming convention. Let's try and stick to PEP8 convention for Class names (CamelCase)

https://peps.python.org/pep-0008/

Reply:
Should be fixed in [fa4ac19] (https://github.com/LHRIC/LapSim/pull/1/commits/fa4ac196dc3ab53955c680b0da7b1fdc27686bec)